packages/db/postgres/tables.sql

```diff
@@ -176,10 +176,16 @@ CREATE TABLE IF NOT EXISTS backup
 
 CREATE INDEX IF NOT EXISTS backup_upload_id_idx ON backup (upload_id);
 
+
+-- Setting search_path to public scope for uuid function(s)
+SET search_path TO public;
+DROP extension IF EXISTS "uuid-ossp";
+CREATE extension "uuid-ossp" SCHEMA public;
+
 -- Tracks requests to replicate content to more nodes.
 CREATE TABLE IF NOT EXISTS pin_request
 (
-  id              BIGSERIAL PRIMARY KEY,
+  id              UUID DEFAULT public.uuid_generate_v4() PRIMARY KEY,
```

Comment:
Nice!
I'm wondering though if we should have a separate field altogether for this to avoid exposing the PK?